e2e/tests/api-driven/package.json

```diff
@@ -6,7 +6,7 @@
   },
   "dependencies": {
     "@cucumber/cucumber": "^9.3.0",
-    "@opensystemslab/planx-core": "git+https://github.com/theopensystemslab/planx-core#2732b6e",
+    "@opensystemslab/planx-core": "git+https://github.com/theopensystemslab/planx-core#58a0647",
```

Comment:
I'll update this cross-project once I've merged https://github.com/theopensystemslab/planx-core/pull/388 to `main`

Reply:
Done ✅